cbcbeat/monodomainsolver.py

```diff
@@ -244,6 +252,7 @@ def default_parameters():
         params = Parameters("BasicMonodomainSolver")
         params.add("theta", 0.5)
         params.add("polynomial_degree", 1)
+        params.add("family", "CG")
```

Comment:
Could we use "Lagrange" instead of "CG"?:) Just a stylistic choice, as CG isn't really a finite element but a method.

Reply:
Sure